src/khoj/processor/tools/online_search.py

```diff
@@ -23,6 +23,10 @@
 SERPER_DEV_API_KEY = os.getenv("SERPER_DEV_API_KEY")
 SERPER_DEV_URL = "https://google.serper.dev/search"
 
+JINA_READER_API_URL = "https://r.jina.ai/"
+JINA_SEARCH_API_URL = "https://s.jina.ai/"
+JINA_API_KEY = os.getenv("JINA_API_KEY")
```

Comment:
Wait, is this required for the integration to work? The documentation suggests that it should work out of the box. 

If there's a scenario were someone might benefit from providing the key, maybe add that in the documentation?